api/engine.py

```diff
@@ -368,9 +457,51 @@ def __get_logs():
         host = __get_value(flask_request, "host")
     except Exception:
```

Comment:
@aman566  please don't leave bare exceptions - please check for other flake8/pylint issues